src/cljs/athens/parse_renderer.cljs

```diff
@@ -15,28 +15,32 @@
   "Transforms Instaparse output to Hiccup."
   [tree]
   (insta/transform
-    {:block      (fn [& contents]
-                   (concat [:span {:class "block"}] contents))
-     :block-link (fn [title]
-                   (let [id (subscribe [:block/uid [:node/title title]])]
-                     [:span {:class "block-link"}
-                      [:span {:style {:color "gray"}} "[["]
-                      [:a {:href  (rfee/href :page {:id (:block/uid @id)})
-                           :style {:text-decoration "none" :color "dodgerblue"}} title]
-                      [:span {:style {:color "gray"}} "]]"]]))
-     :block-ref  (fn [id]
-                   (let [string (subscribe [:block/string [:block/uid id]])]
-                     [:span {:class "block-ref"
-                             :style {:font-size "0.9em" :border-bottom "1px solid gray"}}
-                      [:a {:href (rfee/href :page {:id id})} (parse-and-render (:block/string @string))]]))
-     :hashtag    (fn [tag-name]
-                   (let [id (subscribe [:block/uid [:node/title tag-name]])]
-                     [:a {:class "hashtag"
-                          :style {:color "gray" :text-decoration "none" :font-weight "bold"}
-                          :href  (rfee/href :page {:id (:block/uid @id)})}
-                      (str "#" tag-name)]))
-     :bold       (fn [text]
-                   [:strong {:class "bold"} text])}
+    {:block     (fn [& contents]
+                  (concat [:span {:class "block"}] contents))
+     :page-link (fn [title]
+                  (let [id (subscribe [:block/uid [:node/title title]])]
+                    [:span {:class "page-link"}
+                     [:span {:style {:color "gray"}} "[["]
+                     [:a {:href  (rfee/href :page {:id (:block/uid @id)})
+                          :style {:text-decoration "none" :color "dodgerblue"}} title]
+                     [:span {:style {:color "gray"}} "]]"]]))
+     :block-ref (fn [id]
+                  (let [string (subscribe [:block/string [:block/uid id]])]
+                    [:span {:class "block-ref"
+                            :style {:font-size "0.9em" :border-bottom "1px solid gray"}}
+                     [:a {:href (rfee/href :page {:id id})} (parse-and-render (:block/string @string))]]))
+     :hashtag   (fn [tag-name]
+                  (let [id (subscribe [:block/uid [:node/title tag-name]])]
+                    [:a {:class "hashtag"
+                         :style {:color "gray" :text-decoration "none" :font-weight "bold"}
+                         :href  (rfee/href :page {:id (:block/uid @id)})}
+                     (str "#" tag-name)]))
+     :url-link  (fn [{url :url} text]
+                  [:a {:class "url-link"
+                       :href url}
+                   text])
+     :bold      (fn [text]
+                  [:strong {:class "bold"} text])}
```

Comment:
The only thing I can think of to make this more robust against indenting changes, is extracting the different parts to functions (e.g. `:page-link page-link`)